### ISSUE TITLE
Install.py issue with latest Ansible 2.10 devel version

### ISSUE THREAD

**Issue by matt6697** [citrix/citrix-adc-ansible-modules#117]
The `python install.py` command fails with the following error with latest Ansible 2.10 development sources as the `module_utils/network` and `module/network` directories seems to have been removed in  Ansible 2.10

```
Module utils directory (/<path>/virtualenvironment/lib/python3.6/site-packages/ansible/module_utils/network/netscaler) does not exist
```

**Comment by l3ender** [comment]
See also https://github.com/citrix/citrix-adc-ansible-modules/issues/27.

**Comment by l3ender** [comment]
I pushed a fix for this in https://github.com/citrix/citrix-adc-ansible-modules/pull/131 and I was able to install.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #145 

Modules are installable as collections from the github repository.

See this [section](https://github.com/citrix/citrix-adc-ansible-modules#installation) of the README
